### PR TITLE
Added message for rise cache out of space error

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees could you please review and merge and it is validated? cheers

### CODE REVIEW

File: src/widget/storage-folder.js [hunk 1]
```diff
@@ -148,18 +148,18 @@ RiseVision.Image.StorageFolder = function (data) {
...
 
+      var statusCode = 0;
       // Show a different message if there is a 404 coming from rise cache
-      var statusCode = e.detail.error.currentTarget.status;
-
-      var errorMessage = "There was a problem retrieving the file from Rise Cache.";
-      if(statusCode === 404){
-        errorMessage = "The image does not exist or cannot be accessed.";
+      if(e.detail.error.message){
+        statusCode = +e.detail.error.message.substring(e.detail.error.message.indexOf(":")+2);
```
Comment (stulees) [inline src/widget/storage-folder.js]: @rodrigopavezi Should there be `+` in front of `e.detail...`?

Comment (rodrigopavezi) [inline src/widget/storage-folder.js]: @stulees good question. Yes, it converts the string to an integer for the switch statement. cheers

Comment (stulees) [inline src/widget/storage-folder.js]: @rodrigopavezi Ah right, a unary plus. Makes sense, thanks for clarifying

**Comment by stulees** [comment]
@rodrigopavezi :+1:

**Pull request merged**